### PR TITLE
object.size.reduce

### PR DESCRIPTION
**Pull request by imgoej**
The MCMC-list provided by the output of runjags can easily become huge in terms of bytes. It is therefore not prudent to have that list appear twice in the return output of these functions (once as "Sampling" and once as "mcmc" inside "runjags_object").

I therefore set "mcmc" of "runjags_object" to a string pointing to where is "Sampling" is. If input is a BayLum.list (output of these functions and therefore not a generated DataFile), "mcmc" inside "runjags_object" takes on "Sampling" as its value.

### CODE REVIEW

File: R/AgeS_Computation.R [hunk 1]
```diff
@@ -13,7 +13,7 @@
 #' \code{DATA} contains informations for more than one sample.
 #' If there is stratigraphic relations between samples, informations in DATA must be ordered by order of increasing ages.
 #' See the details section to for more informations.
-#' (2): An object of class "runjags" which is provided by the output of [AgeS_Computation]. When input of class "runjags" is identified, no new JAGS model is created. Instead, the JAGS model specified by the "runjags" object is extended. Useful for when convergence was not originally achieved and a complete restart is not desirable.
+#' (2): An object of class "BayLum.list" which is provided by the output of [AgeS_Computation]. When input of class "BayLum.list" is identified, no new JAGS model is created. Instead, the JAGS model specified by the AgeS_Computation output is extended. Useful for when convergence was not originally achieved and a complete restart is not desirable.
```
Comment (RLumSK) [inline R/AgeS_Computation.R:16]: Please use markdown syntax:
* "BayLum.list" -> \`BayLum.list\` 
* AgeS_Computation -> \[AgeS_Computation\]

The first renders the term correctly in the manual, and the second links to the function.

File: R/Age_OSLC14.R [hunk 2]
```diff
@@ -15,7 +15,7 @@
 #' \code{DATA} contains information for more than one sample.
 #' If there is stratigraphic relations between samples, informations in DATA must be ordered by order of
 #' increasing ages. See the details section to for more informations.
-#' (2): an object of class "runjags" which is provided by the output of [Age_OSLC14]. When input of class "runjags" is identified, no new JAGS model is created. Instead, the JAGS model specified by the "runjags" object is extended. Useful for when convergence was not originally achieved and a complete restart is not desirable.
+#' (2): an object of class "BayLum.list" which is provided by the output of [Age_OSLC14]. When input of class "BayLum.list" is identified, no new JAGS model is created. Instead, the JAGS model specified within the BayLum.list is extended. Useful for when convergence was not originally achieved and a complete restart is not desirable.
```
Comment (RLumSK) [inline R/Age_OSLC14.R:18]: Same as before, please use markdown syntax:

* "BayLum.list" -> \`BayLum.list\`

**Review by RLumSK — CHANGES_REQUESTED** [review]
Please address the two minor comments, then I can merge the PR

**Comment by RLumSK** [comment]
@imgoej I forgot, please also create a NEWS entry in NEWS.Rmd and then render it with `RLumBuild::module_knit_README()`.

File: R/AgeS_Computation.R (hunk 1 above)
Comment (imgoej) [reply R/AgeS_Computation.R]: Done.

**Comment by RLumSK** [comment]
@imgoej Thanks!

**Pull request merged**